osutil/mountprofile_test.go

```diff
@@ -58,6 +58,20 @@ func (s *profileSuite) TestLoadMountProfile2(c *C) {
 	})
 }
 
+// Test that loading profile with various comments works as expected.
+func (s *profileSuite) TestLoadMountProfile3(c *C) {
+	dir := c.MkDir()
+	fname := filepath.Join(dir, "existing")
+	err := ioutil.WriteFile(fname, []byte("  # comment\nname-1 dir-1 type-1 options-1 1 1 # 1st entry  # comment\n\n"), 0644)
```

Comment:
Maybe use `` style here to make this easier to read?

Reply:
Also, should this test for a `#` inside an option? I.e. ensure that in this case it is not parsed as a comment? e.g. dir-1#not-a-comment or something?